python/cuml/tests/test_svm.py

```diff
@@ -451,105 +451,6 @@ def get_memsize(svc):
     return ms
 
 
-@pytest.mark.xfail(reason="Need rapidsai/rmm#415 to detect memleak robustly")
```

Comment:
These tests made use of `handle`, along with many long-since broken APIs. I opted to rip them out rather than just tweak the `handle` bits since it didn't seem worth it to keep around tests we know are broken and aren't running. Can always bring them back later, they're still in the git history.